README.md

```diff
@@ -602,13 +602,18 @@ path = ArtifactoryPath(
 # Get FileStat
 stat = path.stat()
 print(stat)
-print(stat.md5)
+print(ctime)
```

Comment:
Where have we defined `ctime`? I don't see it above, probably it should be `stat.ctime`

Reply:
it is stat.ctime
sorry for that.

Reply:
should be fixed.